games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild

```diff
@@ -18,8 +18,8 @@ RDEPEND="media-libs/libsdl[joystick,opengl,sound,video]
 	sys-libs/zlib
 	virtual/opengl
 	dev-qt/qtcore:4
-	dev-qt/qtgui:4
-	dev-qt/qtopengl:4
+	dev-qt/qtgui:4[-egl]
+	dev-qt/qtopengl:4[-egl]
 	dev-libs/libcdio"
 DEPEND="${RDEPEND}
 	>=sys-devel/gcc-4.4"
```

Comment:
Correct. Though I'm not sure if it's worth the effort for such an ancient version.